client/Combatant/Combatant.ts

```diff
@@ -77,7 +77,7 @@ export class Combatant {
   private processCombatantState(savedCombatant: CombatantState) {
     this.IndexLabel = savedCombatant.IndexLabel || 0;
     this.CurrentHP(savedCombatant.CurrentHP);
-    this.CurrentNotes(savedCombatant.CurrentNotes || "");
+    this.CurrentNotes(savedCombatant.CurrentNotes || this.AutoPopulatedNotes());
```

Comment:
I think this call probably belongs in `AddCombatantFromStatBlock()`, assigned to `initialState.CurrentNotes`. My reasoning is that this should only ever auto-populate when adding from a StatBlock. 

An interesting side effect of putting the call here is that very old savedCombatants (from before CurrentNotes) would receive AutoPopulatedNotes(), but newer ones would not.